src/pages/PictureUploadBox.tsx

```diff
@@ -37,12 +39,29 @@ const PictureUploadBox = ({
   onClickDeleteImage,
   onClickSetMainImage,
   onClickAddImage,
+  onDragAddImage,
   ...props
 }: Props) => {
   const { t } = useTranslation();
 
+  const handleDrop = (e: DragEvent<HTMLElement>) => {
+    e.nativeEvent.preventDefault();
+
+    if (!e) return;
```

Comment:
Shouldn't the early return be above the prevent?

```suggestion
    if (!e) return;
 
    e.nativeEvent.preventDefault();
```